libs/transactions-journey/e2e-tests/specs/transactions-list.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, Route, TestType } from '@playwright/test';
+import { TRANSACTIONS_LIST } from '../data/transactions-list.data';
+import { TransactionsListPage } from '../page-object/transactions-list.page';
+
+export interface TransactionsListFixture {
+  listPage: TransactionsListPage;
+  listData?: typeof TRANSACTIONS_LIST;
+}
+
+export function testTransactionsList(
+  test: TestType<TransactionsListFixture, TransactionsListFixture>
+) {
+  test.describe('Transactions', () => {
+    test.describe('Transactions list', () => {
```

Comment:
@edwardroth is there anything we want to add here as a demonstration from a QA POV? Assuming we want to merge everything into a single golden-sample repo

Reply:
I think the only things I would change are:
- Adding tags. These can be used to easily run subsets of tests, for example   `test(`should display transactions`, { tag: '@e2e', @transactions }, async ({ listPage }) => {`
- It might be useful (but they can look at the example in `apps/golden-sample-app-e2e/specs/login.spec.ts`) to show `test.step` being used.

If more examples are needed we can add these to the backlog. I think this is adequate enough to get people started.

Reply:
And as mentioned in [another comment](https://github.com/Backbase/golden-sample-app/pull/344/files#r1870937256) to remove one of the describe blocks. I think having two of them here makes it more bloated than it needs to be.

Reply:
Do you think the tag should be unique per joruney or should it be unique on the capability level, screen level ?
A few examples:
```
// all the transactions across the app
@transactions
// a specific functionality of the transactions
@transactions-details
// accounts journey, all transactions
@accounts-journey-transactions
// accounts journey, transactions details screen
@accounts-journey-transactions-details
```

Reply:
You can have multiple tags and specify these when you run the tests. I would suggest having a `@transactions` tag as a cover all for all the transactions tests, and then another tag like `@transaction-details` to that specific spec file.

Reply:
Added tags